### PR TITLE
Password reset change

### PR DESCRIPTION
**Pull request by shrutiyer**
Bootstrap form with ability to change password in the settings page

### CODE REVIEW

**Comment by davidabrahams** [comment]
mergey merge merge

**Comment by BrennaManning** [comment]
Tiny tiny change - just changed the admin labels to say admin/staff instead of true/false

**Comment by shrutiyer** [comment]
@nmohamed If everything okay, merge?

File: public/javascripts/ng-controllers.js [hunk 1]
```diff
@@ -59,7 +57,7 @@ springInitiative.controller('indexController', function($scope, $rootScope,
       $state.go('login')
     })
     .error(function(data){
-      console.log("Error")
+      console.log("Error: "+data)
```
Comment (nmohamed) [inline public/javascripts/ng-controllers.js]: semicolon

File: public/javascripts/ng-controllers.js [hunk 2]
```diff
@@ -140,12 +138,13 @@ springInitiative.controller('addStudentController', function($scope, $rootScope,
...
 
 springInitiative.controller('settingsController', function($scope, $rootScope, $http, $location){
+
   $http.get('api/allUsers')
   .success(function(data) {
     $scope.allUsers = data;
   })
   .error(function(data) {
-    console.log('Error:' + data)
+    console.log('Error: ' + data)
```
Comment (nmohamed) [inline public/javascripts/ng-controllers.js]: semicolon

File: public/javascripts/ng-controllers.js [hunk 3]
```diff
@@ -154,7 +153,31 @@ springInitiative.controller('settingsController', function($scope, $rootScope, $
       username.isAdmin = !username.isAdmin;
     })
     .error(function(data) {
-      console.log('Error occured while admin change');
+      console.log('Error: ' + data)
```
Comment (nmohamed) [inline public/javascripts/ng-controllers.js]: semicolon

File: public/javascripts/ng-controllers.js [hunk 4]
```diff
@@ -154,7 +153,31 @@ springInitiative.controller('settingsController', function($scope, $rootScope, $
...
     });
   };
+
+  $scope.changePassword = function(user) {
+    if($scope.currentPassword1 === $scope.currentPassword2 && $scope.currentPassword1 != undefined){
```
Comment (nmohamed) [inline public/javascripts/ng-controllers.js]: whats the difference between 1 and 2? is there a way to make the names clearer?

File: public/javascripts/ng-controllers.js [hunk 5]
```diff
@@ -154,7 +153,31 @@ springInitiative.controller('settingsController', function($scope, $rootScope, $
...
+      $http.post('api/changePassword/'+user._id,{
+        password: $scope.currentPassword1
+      }).success(function(data) {
+        $scope.form_change_password.$setPristine();
+        $scope.currentPassword1 = null;
+        $scope.currentPassword2 = null;
+        $scope.password_match_error = null;
+        $scope.password_change_msg = data.msg;
```
Comment (nmohamed) [inline public/javascripts/ng-controllers.js:169]: for the values above change them to camelcase since we want to be consistent

**Comment by nmohamed** [comment]
:gem:

File: public/javascripts/ng-controllers.js (hunk 4 above)
Comment (shrutiyer) [inline public/javascripts/ng-controllers.js]: The user will be asked to enter password twice in the form. So if they entered the right thing twice, the password changes. Else, enter again.

**Comment by shrutiyer** [comment]
Made changes according to Nora's comments.
Merging it.

**Comment by nmohamed** [comment]
@shrutiyer why did you change it from camelcase for the naming convention? this is `camelCase` while `we_dont_want_this` format
also, as a side note - git etiquette is to not merge your own pull requests

**Comment by shrutiyer** [comment]
@nmohamed It was easier to change it to abc_def. But I can change it to camel case if needed.

**Pull request merged**